### ISSUE TITLE
Running in xcode build lint mode displays warnings but fails builds. Should treat warnings as errors.

### ISSUE THREAD

**Issue by sushant-here** [nicklockwood/SwiftFormat#1433]
Hi,
I am trying to configure swiftformat to run in my code build phases. 

Im running it as `swiftformat target1 target1Tests --swiftversion 5.7.2 --lint`

This produces the following output

`Running SwiftFormat...
(lint mode - no files will be changed.)
Reading config file at target1/.swiftformat
target1/Utilities/SomeFile.swift:78:1: warning: (redundantParens) Remove redundant parentheses.
SwiftFormat completed in 0.17s.
Source input did not pass lint check.
1/312 files require formatting.`

The problem I have is that the linter found a **warning** on line 78 `SomeFile.swift`... xcode displays this as a warning (so its easy to miss). However, the build fails because the swiftformat command return error code `1`.

TLDR... What this means is xcode will cause the build to fail (due to the response code from swiftformat) and it will display a warning (that will get missed by the developer) instead of an error.

From what I have read there is no way to force lint warnings to be treated as errors. Can this please be added as an optional flag? This will allow xcode to display the lint issues as errors (since they are what caused the build the fail).

**Comment by sushant-here** [comment]
From what I can see it is hardcoded as a `warning` in `Formatter.Change.description`... would need to add a command line arg and replace the hardcoded warning with something a bit more... dynamic.

**Comment by nicklockwood** [comment]
You're right, this is confusing. I'll look into the best way to fix it.

**Comment by nicklockwood** [comment]
@sushant-here @bobby-fi fixed in 0.53.5
